README.md

```diff
@@ -52,54 +52,6 @@ preview.show(FILE_ID, ACCESS_TOKEN, {
 ```
 Where `viewerOptions` is an optional object of viewer-specific annotator options and `disabledAnnotationTypes` is an optional string array of valid annotation types to disable. See [Enabling/Disabling Annotations and Annotation Types](#enablingdisabling-annotations-and-annotation-types) below for more details on viewer-specific annotation configurations.
 
-Parameters & Options
```

Comment:
What happens if you create your own `BoxAnnotations` instance to pass into preview, do you not get access to any of these options? Isn't the point that you could se your own conf in that case?

Reply:
The removed section is in relation to instantiating specific Annotators rather than the BoxAnnotations object itself.